docs/ECCTOOL.md

```diff
@@ -1,6 +1,10 @@
 # ECChronos tool
 
```

Comment:
Perhaps we should look into linking to this file from the main README. Perhaps under it's own category.